### PR TITLE
Initial release.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,84 +1,81 @@
...
+Get an API key from https://platform.openai.com/account/api-keys.
 
-Either add a `name: ...` entry to the package.yaml or change the title
-(first line) of this README to the package name.
+We recommend to provide the key through an environment variable on
+desktop machines and through a separate main-file on microcontrollers.
 
-Either add a `description: ...` entry to the package.yaml or ensure
-that the first paragraph of this README can be used as a description.
+For Example, you would have a file `esp32.toit` which imports the `main.toit`
```
Comment (kasperl) [inline README.md]:
```suggestion
For example, you would have a file `esp32.toit` which imports the `main.toit`
```

File: README.md [hunk 2]
```diff
@@ -1,84 +1,81 @@
...
-and
+main args:
+  real_main.main args --key=KEY
 ```
-// Copyright (C) 2022 Jane/John Doe
-// Use of this source code is governed by a Zero-Clause BSD license that can
-// be found in the examples/EXAMPLES_LICENSE file.
+Check this file in.
+
+Locally, make a copy of this file (as `esp32.toit) and insert your API key.
```
Comment (kasperl) [inline README.md]:
```suggestion
Locally, make a copy of this file (as `esp32.toit`) and insert your API key.
```

File: README.md [hunk 3]
```diff
@@ -1,84 +1,81 @@
...
+desktop machines and through a separate main-file on microcontrollers.
 
-Either add a `description: ...` entry to the package.yaml or ensure
-that the first paragraph of this README can be used as a description.
+For Example, you would have a file `esp32.toit` which imports the `main.toit`
+file and provides the API key.
 
-## Structure
-Code that should be used by other developers must live in the `src` folder.
+Typically you would have an `esp32-example.toit` file which looks like this:
```
Comment (kasperl) [inline README.md]:
```suggestion
Typically your `esp32.toit` file would look like this:
```

**Review by kasperl — APPROVED** [review]
Looks good to me.

**Comment by floitsch** [comment]
I ended up removing the openai openapi document. It's relatively big, and available online anyway.

**Pull request merged**